vet.rsc.OpenRSC.Launcher.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>vet.rsc.OpenRSC.Launcher.desktop</id>
```

Comment:
it'd be best if you removed the .desktop here (per the recommendations https://github.com/flathub/flathub/wiki/AppData-Guidelines#id ) and instead included that path (with the .desktop) as a launchable https://github.com/flathub/flathub/wiki/AppData-Guidelines#Launchable